.github/labeler.json

```
@@ -4,11 +4,12 @@
     "area: documentation": ["docs/**", "errors/**"],
     "area: examples": ["examples/**"],
     "area: next/image": [
-      "**/*image*",
-      "**/*image*/**",
+      "packages/*image*",
+      "packages/*image*/**",
```

This won't work right? You want this:
```suggestion
      "packages/**/*image*",
      "packages/**/*image*/**",
```